gremlin-javascript/src/main/javascript/gremlin-javascript/test/cucumber/feature-steps.js

```diff
@@ -20,28 +20,28 @@
 /**
  * @author Jorge Bay Gondra
  */
-'use strict';
 
-const {Given, Then, When, setDefaultTimeout} = require('cucumber');
+import chaiString from 'chai-string';
+import { Given, Then, When, setDefaultTimeout } from '@cucumber/cucumber';
 // Setting Cucumber timeout to 10s for Floating Errors on Windows on GitHub Actions
 setDefaultTimeout(10 * 1000);
-const chai = require('chai')
-chai.use(require('chai-string'));
-const expect = chai.expect;
-const util = require('util');
-const gremlin = require('./gremlin').gremlin;
-const graphModule = require('../../lib/structure/graph');
-const graphTraversalModule = require('../../lib/process/graph-traversal');
-const traversalModule = require('../../lib/process/traversal');
-const utils = require('../../lib/utils');
-const traversal = require('../../lib/process/anonymous-traversal').traversal;
-const Path = graphModule.Path;
-const __ = graphTraversalModule.statics;
-const t = traversalModule.t;
-const P = traversalModule.P;
-const direction = traversalModule.direction;
-const merge = traversalModule.merge;
-const deepMembersById = require('./element-comparison').deepMembersById;
+import { use, expect as _expect } from 'chai';
+use(chaiString);
+const expect = _expect;
+import { inspect, format, inherits } from 'util';
+import { gremlin } from './gremlin.js';
+import { Path as _Path, Vertex, Edge } from '../../lib/structure/graph.js';
```

Comment:
Why not just
`import { Path, Vertex, Edge } from '../../lib/structure/graph.js';` ?

Reply:
Hmm, I used the refactor functionality of VSCode to change these, look like it did something funny with default export. Let me do a quick scan n fix all these.

Reply:
Yep, fixed all the weird auto refactor issues.